CHANGELOG.txt

```diff
@@ -1,22 +0,0 @@
-# CF Revision Manager Changelog
```

Comment:
We can't trash the changelog. In fact, this branch will warrant a new entry to explain the total overhaul that is taking place in what will become version 2.0.0.